modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorage.java

```diff
@@ -118,6 +122,8 @@ public abstract class AbstractPageMemoryMvPartitionStorage implements MvPartitio
 
     private final UpdateTimestampHandler updateTimestampHandler;
 
+    private volatile long estimatedSize;
```

Comment:
It doesn't really make much sense to have this field in abstract class. Persistent implementation can use meta instance for both the access and modifications. Only volatile implementation needs a field. Can you please make these methods abstract?